lib/danger_plugin.rb

```diff
@@ -166,7 +166,22 @@ def find_swift_files(dir_selected, files = nil, excluded_paths = [], included_pa
 
     # Get the configuration file
     def load_config(filepath)
-      filepath ? YAML.load_file(filepath) : {}
+      return {} if filepath.nil? || !File.exist?(filepath)
+
+      config_file = File.open(filepath).read
+
+      # Replace environment variables
+      config_file = parse_environment_variables(config_file)
+
+      YAML.safe_load(config_file)
+    end
+
+    # Find all requested environment variables in the given string and replace them with the correct values.
+    def parse_environment_variables(file_contents)
+      file_contents.gsub(/\$\{([^{}]+)\}/) do |environment_variable|
```

Comment:
For this kind of regexing, it'd be nice to have a few comments describing what this does. Maybe a link out to the docs to read more.

And `environment_variable` is a very... Swift variable name 😅 I'd go with something shorter, like `env_var`.

Reply:
haha, I'm a bit swifty! Thanks for that feedback. I'll add a describing documentation, that makes sense!